test/utils/throwingInstance.ts

```diff
@@ -0,0 +1,24 @@
+import { instance } from "ts-mockito";
+
+/**
+ * This will be useful for detecting when certain methods have not been stubbed.
```

Comment:
Comments should mostly describe a high level of what the function/object does. In this case I would say something like: `Creates an instance of a mocked object. Calls to methods or properties that have not been stubbed will throw errors.`